src/components/export-type.tsx

```diff
@@ -0,0 +1,34 @@
+'use client';
```

Comment:
By default, react components  in next are rendered server side, so the html is rendered while the page is loading and then served to the client as is. In order to use react `useState`, i have to declare this component as a client component that will be rendered client side rather than server side.

That's my understanding of next and this `'use client'` directive. If there's some other way to do this I am happy to adjust.